src/lib/InfiniteScroll.svelte

```diff
@@ -90,4 +78,6 @@
   });
 </script>
 
-<div bind:this={component} style="width:0px" />
+{#if !window && !elementScroll}
+  <div bind:this={component} class="svelte-infinite-scroll" style="width: 0;" />
```

Comment:
what would be the reason for this class?

Reply:
It is just to identify the `svelte-infinite-scroll` component allowing to apply some external style to it. Now the class was replaced by an `id`. 🙂